### ISSUE TITLE
Will HeroDevs maintain this project after August 2023?

### ISSUE THREAD

**Issue by alexsch01** [angular/webdriver-manager#524]
With Chromedriver 115 and above, Chromedriver downloads are moving to https://googlechromelabs.github.io/chrome-for-testing/
- this means ```webdriver-manager update``` will be stuck at Chromedriver 114

When Chrome 116 is released to the Stable channel, ```protractor``` will no longer work

@dwelch2344, will HeroDevs maintain this project?

**Comment by thw0rted** [comment]
If y'all do wind up seeing this issue, can you also please update the README for this project and Protractor as well?  The README files for both repos currently say nothing about EOL status, so people are still participating in issues here.  If nobody is going to answer issues then the repos should be configured to prevent new submissions.  When NPM left Github (for a while), they put their repo in Archived status.

**Comment by alexsch01** [comment]
@thw0rted Instructions to fix it yourself: https://github.com/angular/webdriver-manager/issues/517#issuecomment-1673232896

**Comment by dwelch2344** [comment]
Thanks for the DIY method @alexsch01, and apologies on the delayed response. 

We're still exploring a few options on how we'll tackle issues like this, but what we can commit to is keeping protractor running in good form 🤘

@thw0rted thanks for pointing out the repo-level visibility concerns. I'll raise both points with our team & partners to get addressed asap

**Comment by StanislavKharchenko** [comment]
@dwelch2344 Do you need some contribution helps for upgrade Protractor with latest selenium-webdriver (4.x) treatment?
This approach will eliminate needs to use webdriver-manager, but just rely on selenium-manager which now works and maintain driver updates under the hood.
There are also more things from selenium 4, which is fully w3c and has broader browser supports. Selenium 3 is deprecated (JWP deprecated as well) and some time it will stop working with modern browsers (Protractor 7 will also stop working)

**Comment by dwelch2344** [comment]
@StanislavKharchenko yes! We definitely care and are committed for the long haul of keeping Protractor running, which invariably means staying on top of browsers + tools + etc. 

Would love to chat. Shoot me an email and let's connect? Dave-at-HeroDevs.com

**Comment by VolodymyrDmytriukhin** [comment]
> @thw0rted Instructions to fix it yourself: [#517 (comment)](https://github.com/angular/webdriver-manager/issues/517#issuecomment-1673232896)

Thank you, it works for me
